api/client/commands.go

```diff
@@ -1829,10 +1829,11 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 
 func (cli *DockerCli) CmdLogs(args ...string) error {
 	var (
-		cmd    = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
-		follow = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
-		times  = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
-		tail   = cmd.String([]string{"-tail"}, "all", "Output the specified number of lines at the end of logs (defaults to all logs)")
+		cmd      = cli.Subcmd("logs", "CONTAINER", "Fetch the logs of a container", true)
+		follow   = cmd.Bool([]string{"f", "-follow"}, false, "Follow log output")
+		times    = cmd.Bool([]string{"t", "-timestamps"}, false, "Show timestamps")
+		tail     = cmd.String([]string{"-tail"}, "all", "Output the specified number of lines at the end of logs (defaults to all logs)")
+		truncate = cmd.Bool([]string{"-truncate"}, false, "Fetch and truncate logs of a container")
```

Comment:
`--truncate                Fetch and truncate logs of a container` isn't really what `--truncate` does -

 can you change this flag's help text to just say that it truncates the logs _after_ the server has sent the current log.

I presume the server will do it, even if the client has trouble receiving?